main/exercise/MultipleAnswerDropdown.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/* For licensing terms, see /license.txt */
+
+class MultipleAnswerDropdown extends Question
+{
+    public $typePicture = 'mcma_dropdown.png';
+    public $explanationLangVar = 'MultipleAnswerDropdown';
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->type = MULTIPLE_ANSWER_DROPDOWN;
+    }
+
+    public function createForm(&$form, $exercise)
+    {
+        global $text;
+
+        parent::createForm($form, $exercise);
+
+        $objExe = ChamiloSession::read('objExercise');
+
+        $form->addTextarea(
+            'list_text',
+            [get_lang('AnswerList'), get_lang('EnterListOfAnswersOneAnswerByLine)')],
```

Comment:
It seems there is an extra ) at the end of the variable name that should not be there.